examples/tv-app/android/include/application-basic/ApplicationBasicManager.h

```diff
@@ -32,4 +32,6 @@ class ApplicationBasicManager : public ApplicationBasicDelegate
     uint16_t HandleGetProductId() override;
     CHIP_ERROR HandleGetApplicationVersion(AttributeValueEncoder & aEncoder) override;
     CHIP_ERROR HandleGetAllowedVendorList(AttributeValueEncoder & aEncoder) override;
+
+    std::list<uint16_t> GetAllowedVendorList() override { return { HandleGetVendorId(), 456 }; };
```

Comment:
what is special about 456? Please add a constant where name should be inituitive (e.g. kTestVendorId or similar)

Reply:
If this is meant to be a test vendor id, it should use one of our spec-defined test values....

Reply:
This was returned by the code before this change, and I was worried I would break the CI tests. Let me try removing it...

Reply:
can't remove it or the CI tests fail.